%  LaTeX support: latex@mdpi.com 
%  For support, please attach all files needed for compiling as well as the log file, and specify your operating system, LaTeX version, and LaTeX editor.

%=================================================================
\documentclass[jox,review,submit,pdftex,moreauthors]{Definitions/mdpi} 
%\documentclass[preprints,article,submit,pdftex,moreauthors]{Definitions/mdpi} 
% For posting an early version of this manuscript as a preprint, you may use "preprints" as the journal. Changing "submit" to "accept" before posting will remove line numbers.

% Below journals will use APA reference format:
% admsci, aieduc, behavsci, businesses, econometrics, economies, education, ejihpe, famsci, games, humans, ijcs, ijfs, journalmedia, jrfm, languages, psycholint, publications, tourismhosp, youth

% Below journals will use Chicago reference format:
% arts, genealogy, histories, humanities, jintelligence, laws, literature, religions, risks, socsci

%--------------------
% Class Options:
%--------------------
%----------
% journal
%----------
% Choose between the following MDPI journals:
% accountaudit, acoustics, actuators, addictions, adhesives, admsci, adolescents, aerobiology, aerospace, agriculture, agriengineering, agrochemicals, agronomy, ai, air, algorithms, allergies, alloys, amh, analytica, analytics, anatomia, anesthres, animals, antibiotics, antibodies, antioxidants, applbiosci, appliedchem, appliedmath, appliedphys, applmech, applmicrobiol, applnano, applsci, aquacj, architecture, arm, arthropoda, arts, asc, asi, astronomy, atmosphere, atoms, audiolres, automation, axioms, bacteria, batteries, bdcc, behavsci, beverages, biochem, bioengineering, biologics, biology, biomass, biomechanics, biomed, biomedicines, biomedinformatics, biomimetics, biomolecules, biophysica, biosensors, biosphere, biotech, birds, blockchains, bloods, blsf, brainsci, breath, buildings, businesses, cancers, carbon, cardiogenetics, catalysts, cells, ceramics, challenges, chemengineering, chemistry, chemosensors, chemproc, children, chips, cimb, civileng, cleantechnol, climate, clinbioenerg, clinpract, clockssleep, cmd, cmtr, coasts, coatings, colloids, colorants, commodities, complications, compounds, computation, computers, condensedmatter, conservation, constrmater, cosmetics, covid, crops, cryo, cryptography, crystals, csmf, ctn, curroncol, cyber, dairy, data, ddc, dentistry, dermato, dermatopathology, designs, devices, diabetology, diagnostics, dietetics, digital, disabilities, diseases, diversity, dna, drones, dynamics, earth, ebj, ecm, ecologies, econometrics, economies, education, eesp, ejihpe, electricity, electrochem, electronicmat, electronics, encyclopedia, endocrines, energies, eng, engproc, ent, entomology, entropy, environments, epidemiologia, epigenomes, esa, est, famsci, fermentation, fibers, fintech, fire, fishes, fluids, foods, forecasting, forensicsci, forests, fossstud, foundations, fractalfract, fuels, future, futureinternet, futureparasites, futurepharmacol, futurephys, futuretransp, galaxies, games, gases, gastroent, gastrointestdisord, gastronomy, gels, genealogy, genes, geographies, geohazards, geomatics, geometry, geosciences, geotechnics, geriatrics, glacies, grasses, greenhealth, gucdd, hardware, hazardousmatters, healthcare, hearts, hemato, hematolrep, heritage, higheredu, highthroughput, histories, horticulturae, hospitals, humanities, humans, hydrobiology, hydrogen, hydrology, hygiene, idr, iic, ijerph, ijfs, ijgi, ijmd, ijms, ijns, ijpb, ijt, ijtm, ijtpp, ime, immuno, informatics, information, infrastructures, inorganics, insects, instruments, inventions, iot, j, jal, jcdd, jcm, jcp, jcs, jcto, jdad, jdb, jeta, jfb, jfmk, jimaging, jintelligence, jlpea, jmahp, jmmp, jmms, jmp, jmse, jne, jnt, jof, joitmc, joma, jop, jor, journalmedia, jox, jpbi, jpm, jrfm, jsan, jtaer, jvd, jzbg, kidney, kidneydial, kinasesphosphatases, knowledge, labmed, laboratories, land, languages, laws, life, lights, limnolrev, lipidology, liquids, literature, livers, logics, logistics, lubricants, lymphatics, machines, macromol, magnetism, magnetochemistry, make, marinedrugs, materials, materproc, mathematics, mca, measurements, medicina, medicines, medsci, membranes, merits, metabolites, metals, meteorology, methane, metrics, metrology, micro, microarrays, microbiolres, microelectronics, micromachines, microorganisms, microplastics, microwave, minerals, mining, mmphys, modelling, molbank, molecules, mps, msf, mti, multimedia, muscles, nanoenergyadv, nanomanufacturing, nanomaterials, ncrna, ndt, network, neuroglia, neurolint, neurosci, nitrogen, notspecified, nursrep, nutraceuticals, nutrients, obesities, oceans, ohbm, onco, oncopathology, optics, oral, organics, organoids, osteology, oxygen, parasites, parasitologia, particles, pathogens, pathophysiology, pediatrrep, pets, pharmaceuticals, pharmaceutics, pharmacoepidemiology, pharmacy, philosophies, photochem, photonics, phycology, physchem, physics, physiologia, plants, plasma, platforms, pollutants, polymers, polysaccharides, populations, poultry, powders, preprints, proceedings, processes, prosthesis, proteomes, psf, psych, psychiatryint, psychoactives, psycholint, publications, purification, quantumrep, quaternary, qubs, radiation, reactions, realestate, receptors, recycling, regeneration, religions, remotesensing, reports, reprodmed, resources, rheumato, risks, robotics, rsee, ruminants, safety, sci, scipharm, sclerosis, seeds, sensors, separations, sexes, signals, sinusitis, siuj, skins, smartcities, sna, societies, socsci, software, soilsystems, solar, solids, spectroscj, sports, standards, stats, std, stresses, surfaces, surgeries, suschem, sustainability, symmetry, synbio, systems, tae, targets, taxonomy, technologies, telecom, test, textiles, thalassrep, therapeutics, thermo, timespace, tomography, tourismhosp, toxics, toxins, transplantology, transportation, traumacare, traumas, tropicalmed, universe, urbansci, uro, vaccines, vehicles, venereology, vetsci, vibration, virtualworlds, viruses, vision, waste, water, wem, wevj, wild, wind, women, world, youth, zoonoticdis

%---------
% article
%---------
% The default type of manuscript is "article", but can be replaced by: 
% abstract, addendum, article, benchmark, book, bookreview, briefcommunication, briefreport, casereport, changes, clinicopathologicalchallenge, comment, commentary, communication, conceptpaper, conferenceproceedings, correction, conferencereport, creative, datadescriptor, discussion, entry, expressionofconcern, extendedabstract, editorial, essay, erratum, fieldguide, hypothesis, interestingimages, letter, meetingreport, monograph, newbookreceived, obituary, opinion, proceedingpaper, projectreport, reply, retraction, review, perspective, protocol, shortnote, studyprotocol, supfile, systematicreview, technicalnote, viewpoint, guidelines, registeredreport, tutorial,  giantsinurology, urologyaroundtheworld
% supfile = supplementary materials

%----------
% submit
%----------
% The class option "submit" will be changed to "accept" by the Editorial Office when the paper is accepted. This will only make changes to the frontpage (e.g., the logo of the journal will get visible), the headings, and the copyright information. Also, line numbering will be removed. Journal info and pagination for accepted papers will also be assigned by the Editorial Office.

%------------------
% moreauthors
%------------------
% If there is only one author the class option oneauthor should be used. Otherwise use the class option moreauthors.

%---------
% pdftex
%---------
% The option pdftex is for use with pdfLaTeX. Remove "pdftex" for (1) compiling with LaTeX & dvi2pdf (if eps figures are used) or for (2) compiling with XeLaTeX.

%=================================================================
% MDPI internal commands - do not modify
\firstpage{1} 
\makeatletter 
\setcounter{page}{\@firstpage} 
\makeatother
\pubvolume{1}
\issuenum{1}
\articlenumber{0}
\pubyear{2025}
\copyrightyear{2025}
%\externaleditor{Firstname Lastname} % More than 1 editor, please add `` and '' before the last editor name
\datereceived{ } 
\daterevised{ } % Comment out if no revised date
\dateaccepted{ } 
\datepublished{ } 
%\datecorrected{} % For corrected papers: "Corrected: XXX" date in the original paper.
%\dateretracted{} % For retracted papers: "Retracted: XXX" date in the original paper.
\hreflink{https://doi.org/} % If needed use \linebreak
%\doinum{}
%\pdfoutput=1 % Uncommented for upload to arXiv.org
%\CorrStatement{yes}  % For updates
%\longauthorlist{yes} % For many authors that exceed the left citation part

%=================================================================
% Add packages and commands here. The following packages are loaded in our class file: fontenc, inputenc, calc, indentfirst, fancyhdr, graphicx, epstopdf, lastpage, ifthen, float, amsmath, amssymb, lineno, setspace, enumitem, mathpazo, booktabs, titlesec, etoolbox, tabto, xcolor, colortbl, soul, multirow, microtype, tikz, totcount, changepage, attrib, upgreek, array, tabularx, pbox, ragged2e, tocloft, marginnote, marginfix, enotez, amsthm, natbib, hyperref, cleveref, scrextend, url, geometry, newfloat, caption, draftwatermark, seqsplit
% cleveref: load \crefname definitions after \begin{document}

%=================================================================
% Please use the following mathematics environments: Theorem, Lemma, Corollary, Proposition, Characterization, Property, Problem, Example, ExamplesandDefinitions, Hypothesis, Remark, Definition, Notation, Assumption
%% For proofs, please use the proof environment (the amsthm package is loaded by the MDPI class).

%=================================================================
% Full title of the paper (Capitalized)
\Title{Experimental and in silico aproaches to study CES substrate selectivity}

% MDPI internal command: Title for citation in the left column
\TitleCitation{Experimental and in silico aproaches to study CES substrate selectivity}

% Author Orchid ID: enter ID or remove command
\newcommand{\orcidauthorA}{0000-0002-5626-6719} % Add \orcidA{} behind the author's name
%\newcommand{\orcidauthorB}{0000-0000-0000-000X} % Add \orcidB{} behind the author's name

% Authors, for the paper (add full first names)
\Author{Sergio R. Ribone $^{1}$\orcidA{}* and Mario A. Quevedo $^{1}$}

%\longauthorlist{yes}

% MDPI internal command: Authors, for metadata in PDF
\AuthorNames{Sergio R. Ribone and Mario A. Quevedo}

% MDPI internal command: Authors, for citation in the left column, only choose below one of them according to the journal style
% If this is a Chicago style journal 
% (arts, genealogy, histories, humanities, jintelligence, laws, literature, religions, risks, socsci): 
% Lastname, Firstname, Firstname Lastname, and Firstname Lastname.

% If this is a APA style journal 
% (admsci, behavsci, businesses, econometrics, economies, education, ejihpe, games, humans, ijfs, journalmedia, jrfm, languages, psycholint, publications, tourismhosp, youth): 
% Lastname, F., Lastname, F., \& Lastname, F.

% If this is a ACS style journal (Except for the above Chicago and APA journals, all others are in the ACS format): 
% Lastname, F.; Lastname, F.; Lastname, F.
\isAPAStyle{%
       \AuthorCitation{Ribone, S.R. \& Quevedo, M.A.}
         }{%
        \isChicagoStyle{%
        \AuthorCitation{Ribone, S.R. and Quevedo, M.A.}
        }{
        \AuthorCitation{Ribone, S.R., Quevedo, M.A.}
        }
}

% Affiliations / Addresses (Add [1] after \address if there is only one affiliation.)
\address{%
$^{1}$ \quad Universidad Nacional de C\'ordoba. Facultad de Ciencias Qu\'imicas. Departamento de Ciencias Farmac\'euticas. Consejo Nacional de Investigaciones Cient\'ificas y T\'ecnicas (CONICET), Unidad de Investigaci\'on y Desarrollo en Tecnolog\'ia Farmac\'eutica (UNITEFA), C\'ordoba (X5000HUA), Argentina.\\ 
}

% Contact information of the corresponding author
\corres{Correspondence: ; Tel.: (optional; include country code; if there are multiple corresponding authors, add author initials) +xx-xxxx-xxx-xxxx (F.L.)}

% Current address and/or shared authorship
%\firstnote{Current address: Affiliation.}  
% Current address should not be the same as any items in the Affiliation section.

%\secondnote{These authors contributed equally to this work.}
% The commands \thirdnote{} till \eighthnote{} are available for further notes.

%\simplesumm{} % Simple summary

%\conference{} % An extended version of a conference paper

% Abstract (Do not insert blank lines, i.e. \\) 
\abstract{A single paragraph of about 200 words maximum. For research articles, abstracts should give a pertinent overview of the work. We strongly encourage authors to use the following style of structured abstracts, but without headings: (1) Background: place the question addressed in a broad context and highlight the purpose of the study; (2) Methods: describe briefly the main methods or treatments applied; (3) Results: summarize the article's main findings; (4) Conclusions: indicate the main conclusions or interpretations. The abstract should be an objective representation of the article, it must not contain results which are not presented and substantiated in the main text and should not exaggerate the main conclusions.}

% Keywords
\keyword{keyword 1; keyword 2; keyword 3 (List three to ten pertinent keywords specific to the article; yet reasonably common within the subject discipline.)} 

% The fields PACS, MSC, and JEL may be left empty or commented out if not applicable
%\PACS{J0101}
%\MSC{}
%\JEL{}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Only for the journal Diversity
%\LSID{\url{http://}}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Only for the journal Applied Sciences
%\featuredapplication{Authors are encouraged to provide a concise description of the specific application or a potential application of the work. This section is not mandatory.}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Only for the journal Data
%\dataset{DOI number or link to the deposited data set if the data set is published separately. If the data set shall be published as a supplement to this paper, this field will be filled by the journal editors. In this case, please submit the data set as a supplement.}
%\datasetlicense{License under which the data set is made available (CC0, CC-BY, CC-BY-SA, CC-BY-NC, etc.)}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Only for the journal BioTech, Fishes, Neuroimaging and Toxins
%\keycontribution{The breakthroughs or highlights of the manuscript. Authors can write one or two sentences to describe the most important part of the paper.}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Only for the journal Encyclopedia
%\encyclopediadef{For entry manuscripts only: please provide a brief overview of the entry title instead of an abstract.}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Only for the journal Advances in Respiratory Medicine, Future, Sensors and Smart Cities
%\addhighlights{yes}
%\renewcommand{\addhighlights}{%
%
%\noindent This is an obligatory section in ``Advances in Respiratory Medicine'', ``Future'', ``Sensors'' and ``Smart Cities”, whose goal is to increase the discoverability and readability of the article via search engines and other scholars. Highlights should not be a copy of the abstract, but a simple text allowing the reader to quickly and simplified find out what the article is about and what can be cited from it. Each of these parts should be devoted up to 2~bullet points.\vspace{3pt}\\
%\textbf{What are the main findings?}
% \begin{itemize}[labelsep=2.5mm,topsep=-3pt]
% \item First bullet.
% \item Second bullet.
% \end{itemize}\vspace{3pt}
%\textbf{What is the implication of the main finding?}
% \begin{itemize}[labelsep=2.5mm,topsep=-3pt]
% \item First bullet.
% \item Second bullet.
% \end{itemize}
%}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{document}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Introduction}

Human Carboxylesterases (CES, EC 3.1.1.1) are serine hydrolase enzymes responsible for the metabolism and biotransformation of diverse endogenous and xenobiotic substrates containing esters, thioesters, amides, carbonates
and carbamates moieties in their structures\cite{Wang2018a,Fukami2012,Hosokawa2008}. Based on amino acid sequence 
homology, human CES are classified into five isoforms (CES1–CES5). Among these, CES1 and CES2 are the most 
clinically relevant isoforms\cite{CaseyLaizure2013,Dai2020,Xu2016}.

CES1 and CES2 share 47\% of protein sequence identity but exhibit distinct substrate 
specificities and tissue distributions. Specifically, CES1 is primarily expressed in the liver,
whereas CES2 is predominantly found in the intestines\cite{Wang2018a,CaseyLaizure2013,
Hosokawa2008,Di2019}.  It is widely accepted in the literature that the
substrate binding and hydrolysis selectivity of each isoform are primarily determined
by the size of the acyl and alkyl moieties of the respective
substrate molecular structures\cite{Di2019,CaseyLaizure2013,Xu2016,Fukami2015}.
In this context, CES1 preferentially catalyzes the hydrolysis of substrates with
smaller alkyl than acyl groups, as seen with drugs and prodrugs like clopidogrel, 
oseltamivir and meperidine\cite{Zhang2014,Fukami2015,Fukami2012}. Conversely, CES2 
tends to hydrolyze compounds with smaller acyl than alkyl groups, examples of which include
halperidol, procaine and flutamide\cite{Wang2018a,Fukami2015,Hosokawa2008}. 


Despite this general trend, several exceptions to substrate selectivity have been reported.
Notably, drugs and prodrugs like irinotecan, propanil, oxybutinin and 
prolocaine have been shown to be metabolized by both CES isoforms with similar efficiency
\cite{Fukami2015,Honda2021}. Another noteworthy example is the differential CES-mediated 
biotransformation rates of pyrethroid derivatives, where \textit{cis} and 
\textit{trans} isomers of permethrin exhibit distinct hydrolysis pattern by CES1 and CES2, 
despite having identical acyl and alkyl group sizes\cite{Yang2009}.

Given the critical role of CES-mediated catalysis in various physiological processes, 
understanding the structural determinants of substrate specificity among CES isoforms is
crucial for several areas of research. One such area is the design of prodrugs with 
optimized biopharmaceutical profiles. This approach aims to avoid undesirable early 
biotransformation of the prodrug or to enable site-specific bioactivation targeting 
a particular CES isoform\cite{Pratt2013,Fukami2012}. In addition, the development of specific
inhibitors for CES1 and CES2 offers promising therapeutic potential for managing
metabolic diseases. For instance, the CES1 inhibitor GR148672X has been explored
as a candidate for treating hypertriglyceridemia, obesity and atherosclerosis\cite{Bachovchin2012}.

In the field of medical imaging, there is growing interest in the development of
fluorescent biological probes for selective detection of CES1 and CES2
activity\cite{Zhang2021,Elkhanoufi2022,Dai2021,Dai2020,Jia2021}. Over recent years, 
the selective imaging of \textit{in vivo} enzyme activity has emerged as a powerful method 
for the study of biological systems, due to its ability to provide real-time, noninvasive monitoring 
within living organisms\cite{Zhang2021,Elkhanoufi2022,Dai2021,Dai2020,Jia2021}. In this context, 
the design of fluorescent probes that specifically target CES1 and CES2 offers a promising strategy
for visualizing their activity in complex biological systems.

Enzymology has long been a foundational field for studying enzyme structure 
and function, as well as advancing our understanding of biological phenomena 
such as intermediary metabolism, molecular biology, and cellular signaling 
and regulation\cite{Punekar2025}. Early enzymology focused primarily on experimental
techniques aimed at analyzing the catalytic properties and molecular specificity of 
enzymes\cite{Punekar2025}. The first one studies the thermodynamic and kinetic of the 
enzymatic reaction, measuring free-energy of reaction and activation. The second studies
the specificity of different molecules for the enzymes, not only limited to the substrates, 
but also any other molecule that satisfy the specificity criteria for the enzyme, 
as is the case for potential inhibitors\cite{Punekar2025}.

Due to the complexity of enzymes and the challenges associated with studying 
biomolecular reactions, many questions and mechanisms remain unclear. 
Computational enzymology, defined as the study of enzymes
and the their reactions mechanisms by molecular modeling and simulation
\cite{VanDerKamp2013}, has the unique potential to investigate the dynamic behavior
and reactions of biomolecules at atomic resolution. This approach can address 
unresolved issues by complementing and interpreting findings from experimental 
enzymology\cite{VanDerKamp2013,Lonsdale2010,Lodola2012}. 

Since the beginning in 1976, with the pioneering work of Warshel and Levitt 
(Nobel laureates in 2013), computational enzymology has rapidly evolved over 
the past two decades. This progress has been driven by close collaboration between 
experimental and computational enzymologists, enhancing our ability to explain and 
interpret experimental data\cite{VanDerKamp2013,Lonsdale2010,Lodola2012}. 

Building on the symbiotic relationship between experimental and computational 
enzymology, this review will focus on the recent advancements in methodologies
developed to study substrate selectivity between the two most important isoforms of 
human carboxylesterase: CES1 and CES2.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Part I: Experimental enzymology}

\subsection{Kinetic parameters}

During the study of the enzymatic properties of different substrates, 
there are important kinetic parameters determine by the Michaelis-Menten 
fundamental equation of enzyme kinetics\cite{Punekar2025,Michaelis1913}.
The first parameter is the \textbf{dissociation constant}, also known as
Michaelis constant (K\textsubscript{M}), which reflects the affinity between 
the enzyme and the corresponding substrate. A lower the K\textsubscript{M}
value indicates stronger binding within the enzyme-substrate complex
\cite{Punekar2025,Nishi2006,Fukami2012,Ribone2025}.
The second key kinetic parameter is the \textbf{turnover number}, also referred
as catalytic constant (\textit{k\textsubscript{cat}}). This value represents the
number of catalytic cycles the enzyme can complete per unit of time when it is
fully saturated with substrate. A higher \textit{k\textsubscript{cat}} value 
signifies greater substrate turnover and more efficient metabolism by the enzyme
\cite{Punekar2025,Nishi2006,Fukami2012,Ribone2025}.

The ratio between these two kinetic parameters (\textit{k\textsubscript{cat}}
/K\textsubscript{M}) is known as \textbf{specificity constant},
as it reflects the ability of an enzyme to discriminate between structurally
similar substrates. A higher \textit{k\textsubscript{cat}}/K\textsubscript{M} 
ratio indicates that the enzyme shows high affinity (low K\textsubscript{M} value)
and high catalytic rate for the substrate. Because of this, the specificity constant
is often used as a measure of \textbf{catalytic efficiency}, indicating if the 
ligand is a good or poor substrate for the enzyme\cite{Punekar2025,Ribone2025}.

To obtain these kinetic parameters to quantify the substrate 
selectivity for a specific enzyme, experimental enzymology is used
to measured the progress of an enzyme-catalyzed reaction. Like any other
chemical reaction, the progress can be monitored either by measuring
the formation of the product or the consumption of the substrate. 
Reliable detection methods for product formation or substrate depletion
are essential for a successful enzyme assay\cite{Punekar2025}. 

\subsection{Enzyme sources} 

This section outlines the different CES enzymatic sources used in various 
analytical methodologies for calculating kinetic parameters. The two primary 
sources are \textit{ex vivo} tissues, including purified human tissues and 
human microsomes, and pure recombinant enzymes (Table~\ref{tab1}).

\begin{table}[H] 
	\small % Change table font size
	\caption{Enzymatic sources and analytical methods used for the exploration of CES kinetic parameters on different substrates.\label{tab1}}
	\isPreprints{\centering}{} % Only used for preprints
	\begin{tabularx}{\textwidth}{CCCCCC}
		\toprule
		\textbf{Substrates}	& \multicolumn{2}{c}{\textbf{\textit{Ex vivo} tissue}} & \textbf{Recombinant enzyme} & \textbf{Analytical method} & \textbf{Reference}\\
			& \textit{Human tissue} & \textit{Human miocrosomes} & \\
		\midrule
		Irinotecan	& Liver	& - & - & HPLC-FL & \cite{Humerickhouse2000,Sanghani2004}\\
		Methylphenidate	& Liver	& - & CES1/CES2 & LC/MS & \cite{Sun2004}\\
		Oseltamivir	& -	& HLM/HIM & - & HPLC-UV & \cite{Shi2006,Fukami2015}\\
		Temocapril	& -	& HLM/HIM & CES1/CES2 & HPLC-UV & \cite{Imai2006,Fukami2015}\\
		Aspirin	& -	& HLM/HIM & CES1/CES2 & HPLC-UV & \cite{Imai2006,Tang2006}\\
		Clopidogrel	& -	& HLM/HIM & CES1/CES2 & HPLC-UV & \cite{Tang2006,Fukami2015}\\
		Flutamide & -	& HLM/HIM & CES1/CES2 & HPLC-UV & \cite{Watanabe2009,Kobayashi2012}\\
		Pyrethroids & -	& - & CES1/CES2 & FL & \cite{Nishi2006,Ross2006}\\
		Fluorescein diacetate & -	& HLM/HIM & CES1/CES2 & FL & \cite{Wang2011}\\
		Plasugrel & -	& - & CES1/CES2 & LC/MS & \cite{Williams2008}\\
		Heroin & -	& - & CES1/CES2 & HPLC-UV & \cite{Hatfield2010}\\
		Cocaine & -	& - & CES1/CES2 & HPLC-UV & \cite{Hatfield2010,Yao2018}\\
		Oxybutynin & -	& HLM & CES1/CES2 & LC/MS & \cite{Sato2012}\\
		Prilocaine & -	& HLM & CES1/CES2 & HPLC-UV & \cite{Higuchi2013}\\
		Lidocaine & -	& HLM & CES1/CES2 & HPLC-UV & \cite{Higuchi2013}\\
		Clofibrate & -	& - & CES1/CES2 & HPLC-UV & \cite{Fukami2015}\\
		Fenofibrate & -	& HLM/HIM & CES1/CES2 & HPLC-UV & \cite{Fukami2015,Li2023}\\
		Imidapril & -	& - & CES1/CES2 & HPLC-UV & \cite{Fukami2015}\\
		Enalapril & -	& HLM & CES1/CES2 & LC/MS & \cite{Thomsen2014}\\
		Sacubitril & Liver & - & CES1/CES2 & LC/MS & \cite{Shi2016}\\
		Anordrin & - & HLM/HIM & CES1/CES2 & LC/MS & \cite{Jiang2018}\\
		\bottomrule
	\end{tabularx}
	
	\noindent{\footnotesize{HLM: Human liver microsomes. HIM: human intestine microsomes. FL: Fluorescence}}
\end{table}


\subsubsection{\textit{Ex vivo} tissue}

Since the early 2000's, the enzymatic studies of CES1 and CES2 substrate
hydrolysis have relied on purified isoforms sourced from human liver. 
In several studies, human liver tissue is processed by homogenization
and centrifugation, followed by separation of CES1 and CES2 isoforms using
various chromatographic columns\cite{Humerickhouse2000,Sanghani2004,Sun2004}.
Using this method, it was determined that the produg irinotecan, along with 
other metabolites, is primarily bioactivated by human CES2 isoform
\cite{Humerickhouse2000,Sanghani2004}. 

The second source of CES is human tissue microsomes, which are small
vesicles derived from fragmented cell membranes, mainly endoplasmic 
reticulum. These human microsomes can be obtained by differential 
centrifugation of the corresponding tissue or purchased from
different biological supply companies. Human liver microsomes (HLM) 
are commonly used as enzyme source for measuring metabolic stability, as 
they contain key metabolizing enzymes, including CES. As mentioned in the
introduction, CES1 is predominantly expressed in the liver, making HLM
a valuable source for studies of CES1 substrate selectivity.
Following similar tissue distribution patterns, human intestines microsomes 
(HIM) have been used as a source for CES2 in selectivity assays.
Using this approach, studies have shown that the antiviral produgs oseltamivir 
and temocapril are preferentially activated by CES1 (HLM) over CES2 (HIM)\cite{Shi2006,Imai2006}.

\subsubsection{Recombinant enzyme}

Shortly after the use of human tissue as enzyme source, it became evident that
a more purified version of both CES isoforms were necessary to performed
accurate selectivity enzymatic experiments with different substrates.
Morton and Potter developed a method for cloning and expressing CES
using baculovirus to infect \textit{Spodoptera frugiperda} insect cells
\cite{Morton2000}. This technique enabled the production of recombinant CES1
and CES2, which were then used to measure the enzymatic properties of several
substrates (Table\ref{tab1}). The hydrolysis of pyrethroids by human
CES1 and CES2 have been studied using this recombinant enzyme source
\cite{Nishi2006,Ross2006}. Additionally, the hydrolysis specificity of 
CES1 and CES2 for drugs of abuse, such as heroin and cocaine\cite{Hatfield2010},
as well as other drugs and prodrugs, have also been studied using
recombinant enzymes (Table\ref{tab1})\cite{Sato2012,Higuchi2013,Fukami2015}.

As recombinant CES enzymes became commercially available from multiple suppliers, 
research groups were able to continue exploring the metabolism selectivity of CES 
isoforms across different substrates. For example, studies on the angiotensin-converting
enzyme inhibitors enalapril and ramipril showed that both drugs were selectively 
hydrolyzed by CES1\cite{Thomsen2014}. 
Recombinant CES enzymes have also been used to determine the predominant role of 
CES isoforms in the human bioactivation of prodrugs such as sacubitril\cite{Shi2016}
and anordrin\cite{Jiang2018}. Additionally, recombinant CES enzyme have
facilitated preclinical evaluations of the bioactivation rates for
structurally designed prodrugs, including atorvastatin\cite{Mizoi2016,
Takahashi2025}, indomethacin\cite{Takahashi2018,Takahashi2020} and 
haloperidol\cite{Takahashi2019}.

\subsection{Reported analytical methods}

As mentioned earlier, obtaining accurate kinetic parameters requires
a reliable analytical method to quantify the progress of the 
enzyme-catalyzed reaction. In this section, the different analytical 
methods used for the determination of the described kinetic parameters
for different substrates and both CES isoforms will be described.

\subsubsection{Spectrophotometry}
 
Spectrophotometric methods have been widely used to 
quantitatively determine total CES activity, by measuring the
absorbance of \textit{p}-nitrophenol, produced through the 
hydrolysis of \textit{p}-nitrophenyl acetate (pNPA), at
405 nm (Figure~\ref{fig1})\cite{Wadkins2001,Hatfield2010,Boonyuen2015}. 
This approach was applied to the functional characterization of 
recombinant human CES expressed in \textit{E. coli} as an
alternative method for obtaining the human enzyme\cite{Boonyuen2015}.

\begin{figure}[H]
	%\isPreprints{\centering}{} % Only used for preprints
	\includegraphics[width=14.0 cm]{Figure1.png}
	\caption{Hydrolysis of \textit{p}-nitrophenyl acetate (pNPA) by CES to
	produced \textit{p}-nitrophenol.\label{fig1}}
\end{figure} 

The absorbance properties of \textit{p}-nitrophenol have also been
used to evaluate the kinetic parameters of various \textit{p}-nitrophenyl 
esters derivatives with CES1 and CES2\cite{Hatfield2010}.
The authors observed a correlation between the affinity constant
(K\textsubscript{M}) and the calculated water/octanol partition 
coefficients (clogP) values, concluding that the affinity
of the substrates for both CES isoforms is directly related to their
lipophilicity properties\cite{Hatfield2010}.
In addition, kinetic data for naphthyl esters derivatives
were obtained by measuring the formation of naphthol at 230nm
\cite{Wadkins2001}.

This method is simple and rapid, but it has several disadvantages.
One key issue is the potential overlapping interference between 
substrates and products, which complicates experiments
in complex biological systems. Additionally, the method requires
a larger amount of enzyme, as it is performed in a UV cuvette
with a total volume of 1 ml\cite{Boonyuen2015}.

\subsubsection{Fluorescence}

In contrast to the previously described analytical method, 
the fluorescent probe-based approach is not only simple
but also highly selective and sensitive. These probes quantify
CES activity by detecting changes in fluorescence intensity. 
Known as "off-on" fluorescent probes, they initially exhibited 
little or no fluorescence, but the hydrolyzed products release
strong fluorescence in presence of CES. 

This methodology has been used to determine the kinetic parameters of 
pyrethroids-like substrates containing 6-methoxy-2-naphthaldehyde. The
fluorescence of this moiety is measured with an excitation wavelength 
of 330 nm and an emission wavelength of 465 nm\cite{Nishi2006}. In this
study, it was observed that the steroisomeric centers of these derivatives
presented a differential impact on hydrolysis by the two CES isoforms. 
Specifically, the presence of an (\textit{R})-enantiomer carbon 
adjacent to the ester carbonyl carbon resulted in a greater preference
for CES2 hydrolysis than CES1\cite{Nishi2006}. These findings highlight
the importance of the three-dimensional disposition of the substrate 
groups within the catalytic site of the enzyme for CES hydrolysis selectivity.

In another study, fluorescein diacetate was used as a substrate
to assess CES1 and CES2 selectivity. The hydrolysis of fluorescein 
diacetate by the CES enzymes releases fluorescein, which was quantify 
at an excitation wavelength of 483 nm and emission at 525 nm. 
The results indicated that fluorescein diacetate is a selective CES2
substrate (Figure~\ref{fig2})\cite{Wang2011}.

\begin{figure}[H]
	%\isPreprints{}{% This command is only used for ``preprints''.
		\begin{adjustwidth}{-\extralength}{0cm}
			\centering
			%} % If the paper is ``preprints'', please uncomment this parenthesis.
		\subfloat[\centering]{\includegraphics[width=16.0cm]{Figure2a.png}}\\
		%\hfill
		\subfloat[\centering]{\includegraphics[width=13.0cm]{Figure2b.png}}
		%\isPreprints{}{% This command is only used for ``preprints''.
		\end{adjustwidth}
		%} % If the paper is ``preprints'', please uncomment this parenthesis.
	\caption{hydrolysis of fluorescent probe substrates. (\textbf{a}) 
	Fluorescein diacetate. (\textbf{b})  BODIPY ester.\label{fig2}}
\end{figure} 
 

The high fluorescence quantum yield and
photochemical stability of BODIPY dyes make them excellent candidates
for this analytical methodology. As a result, a BODIPY ester was designed 
as a specific substrate for CES1. The acid product formed after 
CES hydrolysis was used to measured the kinetic parameters 
at an excitation wavelength of 505 nm and emission at 560 nm 
(Figure~\ref{fig2})\cite{Ding2019}.

\subsubsection{Chromatography}

Diverse chromatographic methods have been widely used to determined
the kinetic parameters of numerous substrates (e.g., drugs, 
prodrugs, pyrethroids) due to their high sensitivity, accuracy, 
rapid analysis and reproducibility. One additional advantage
is the ability to separate substrate from products for precise 
quantification. The two most commonly used techniques are 
the high performance liquid chromatography (HPLC)
and liquid chromatography-tandem mass spectrometry (LC/MS).

In the case of HPLC, an additional detection technique is required
to quantify the different ligands after chromatographic separation,
with the two previously described methodologies being the most frequently
used. The spectrophotometric methodology usually implement an UV-detector
(HPLC-UV), as most ligands exhibit absorbance in the ultraviolet spectrum.
This methodology has been widely used to study the CES kinetic parameters
of various therapeutic drugs and prodrugs.

To investigate substrate specificity among CES isoforms, a study was conducted
on 13 compounds, including clopidogrel, clofibrate, oseltamivir,
mycophenolate mofetil, procaine and temocapril, among others 
(Table~\ref{tab1}), using HPLC with specific conditions for each metabolite
(e.g., mobile phase, column and UV wavelength)\cite{Fukami2015}. 
This research group also applied the HPLC-UV method to analyzed the
kinetics of other drugs, such as flutamide\cite{Watanabe2009,Kobayashi2012},
prilocaine and lidocaine\cite{Higuchi2013}. Based on the structural characteristics
of these compounds, the authors proposed the already discussed general substrate
selectivity pattern for CES: CES1 preferentially hydrolyzes ligands with 
smaller alkyl than acyl moiety (e.g., clofibrate, lidocaine, temocapril), while 
CES2 favors those with larger alkyl than acyl groups (e.g., flutamide, procaine).

The metabolism of the abuse drugs cocaine and heroine by CES1 and CES2 
was also studied using this methodology\cite{Hatfield2010}. For 
cocaine, hydrolysis was monitored by quantifying the formation
of benzoylecgonine and benzoic acid at 235nm. After incubation
with both CES isoforms, the results showed that cocaine was exclusively
metabolized by CES2, producing benzoic acid and ecgonine methyl ester.
The second potential metabolic pathway, forming benzoylecgonine and
methanol, was not detected\cite{Hatfield2010}. 

For heroine, hydrolysis was assess by monitoring the formation of
6-acetylmorphine, also at 235nm. Both CES isoforms were found to hydrolyzed
heroin, with CES1 exhibiting a higher catalytic efficiency 
(\textit{k\textsubscript{cat}}/K\textsubscript{M}) compared to CES2\cite{Hatfield2010}.
This result represents an exception to the general CES1 substrate 
specificity, as heroin has a larger size alkyl group than acyl 
moiety in its structure.

A different approach was presented in several studies by Takahashi et 
al., in which all the investigated substrates were structurally
diverse indomethacin-derived prodrugs. The formation of indomethacin, 
monitored at 254 nm, was used as the analytical marker to determine the kinetic parameters\cite{Takahashi2018,Takahashi2020,Takahashi2021}. By synthesizing 
prodrugs bearing a variety of alkyl moieties, the authors were able to
investigate how different structural features influence the hydrolysis 
behavior of both CES isoforms. Specifically, they examined: 1) the effect 
of the steric hindrance on the carbon adjacent to the carbonyl group, 2) 
the influence of electron density around the carbonyl group and 3) the 
chiral recognition ability between CES1 and CES2.

Overall, the results indicated that the indomethacin prodrugs were 
mainly hydrolyzed  by CES1, because the drug structure represents 
the acyl group, which in all the cases was larger than the corresponding 
alkyl group. However, prodrugs with aryl-containing alkyl moieties  
exhibited reduced or even no selectivity between CES isoforms, demonstrating
that steric hindrance near the ester carbonyl carbon plays a crucial
role in determining metabolic selectivity.

This research group also conducted similar studies by synthesizing prodrugs
derivatives of haloperidol\cite{Takahashi2019} and atorvastatin
\cite{Takahashi2025}, using the absorbance properties of these drugs as 
detection methods for the HPLC technique.

On the other hand, LC/MS does not required an additional method
for the quantification of the desired kinetic parameters, as both substrate 
and product are detected directly by the mass spectrometer following
their liquid chromatographic separation. This analytical
technique has primarily been used to investigate the metabolic pathway 
of a diverse set of prodrugs.

Interesting results were observed in the hydrolysis study of 
capecitabine, a carbamate prodrug of 5-fluoruracil used in
the treatment of colorectal cancer\cite{Quinney2005}. The study 
showed that capecitabine is hydrolyzed equally by both CES1 and CES2, 
suggesting that substrates containing carbamate groups 
can be metabolized by either isoform, independent of the size of
their acyl or alkyl moieties. Other prodrugs studied include plasugrel, 
which exhibited selectivity for CES2\cite{Williams2008}, sacubitril, 
selectively activated by CES1\cite{Shi2016}, and anordrin, which showed 
similar hydrolysis parameters with both CES isoforms\cite{Jiang2018}.

In a more recent study, the hydrolysis characteristics of cocaine 
by CES1 were re-examined using LC/MS methodology\cite{Yao2018}. 
This study revealed that cocaine is hydrolyzed by CES1, producing 
benzoylecgonine and methanol, which contrast with earlier findings 
using HPLC-UV, where this hydrolysis product was not detected
(Figure~\ref{fig3})\cite{Hatfield2010}. 
These results suggest that LC/MS provides a more accurate analytical approach
for studying cocaine hydrolysis compared to HPLC-UV.

\begin{figure}[H]
	%\isPreprints{\centering}{} % Only used for preprints
	\includegraphics[width=12.0 cm]{Figure3.png}
	\caption{Different obtained cocaine metabolic pathway from CES1 and CES2 
	hydrolysis experimental studies with HPLC-UV and LC/MS analytical methodologies.\label{fig3}}
\end{figure} 


This section highlights the importance of experimental enzymology in 
studying CES substrate selectivity. The gathered information shows that,
despite the general structural rules established for substrate selectivity
between CES1 and CES2, several exceptions and underlying structural 
properties remain unclear. Given that CES binding and hydrolysis involve 
subtle intermolecular interactions, computational enzymology emerges as a 
crucial tool for analyzing CES substrate selectivity at the atomistic level.

\subsection{Experimental CES kinetic parameter databases}

In the era of bioinformatics, providing reliable kinetic parameter data and 
effective data management systems is essential to support researchers in 
retrieving enzymatic information from the vast amount of biological data 
published annually, such as those related to CES enzymes.
In this context, online databases have become invaluable 
tools for granting researchers access to this information. This section
will focus on two of the the most well-known and widely used online 
databases for obtaining human CES enzymatic parameters for different substrates.

\subsubsection{BRENDA database}

The BRaunschweig ENzyme DAtabase (BRENDA) is the oldest collection of 
enzyme-related data from the scientific literature, established in 1987 at
the German National Research Centre for Biotechnology in Braunschweig
\cite{Schomburg2017}. Today, the BRENDA website 
(\url{www.brenda-enzymes.org/}) is accessed by over 100,000 users
each month. The site is very intuitive, allowing users
to search by entering a text, such as enzyme name, ligand name, EC class,
inhibitors, etc., or through structured-based queries by drawing 
substrate/products or ligand substructures\cite{Schomburg2017}.

The search for CES (EC 3.1.1.1) enzymatic information on BRENDA, 
yielded 2079 substrates/products and 67 natural products from the database. 
From these ligands, 849 K\textsubscript{M} values, 550 turnover numbers
(\textit{k\textsubscript{cat}}) and 232 \textit{k\textsubscript{cat}}
/K\textsubscript{M} values (catalytic efficiency) can be retrieved
from diverse bibliographic sources. In addition, the reaction diagrams and 
references associated with these substrates were provided. 

However, two main disadvantage were noted: 1) it is not possible to filtrate 
information based on the organism origin of the enzyme, and 2) the kinetic 
information can not easily be separate by CES isoforms. Despite this
limitations, BRENDA remains a valuable starting point for searchif kinetic 
parameters related to CES1 and CES2 substrate hydrolysis.

\subsubsection{SABIO-RK database}

SABIO-RK is a manually curated database containing enzymatic biochemical
reactions and their kinetic parameters (\url{sabiork.h-its.org/}). The 
authors have established that the database is a valuable resource for both 
experimental and computational enzymology researchers\cite{Wittig2014}.
Data in SABIO-RK are primarily extracted manually from the 
literature and stored in a structured and standardized format. The 
database includes essential data to describe the characteristics 
of biochemical reactions, the corresponding biological source,
kinetic properties and experimental conditions\cite{Wittig2014}.

The regular search for CES name in SABIO-RK website returned 519 entries, 
which is a smaller amount of information compared to BRENDA. The SABIO-RK
website offers an advance search feature that allows filtering by 
various conditions, such as "organisms: \textit{Homo sapiens}", 
which narrows the results to 168 entries related to human CES data. 
Additionally, it is possible to filter the results to include only
data from recombinant CES enzyme (82 entries). However, like BRENDA database,
SABIO-RK also has the limitation of not separating kinetic parameters by CES
isoforms. In conclusion, while SABIO-RK contains less information than BRENDA, 
it offers better organization and more intuitive filtering options, making it 
easier to retrieve relevant data.

\section{Part II: Computational enzymology}

\subsection{Enzyme sources: structures}

At the beginning of any computational molecular modeling 
substrate-enzyme complex study, the information regarding the 
structure of both, the enzyme and the substrate, is needed.
The structures of the ligands are relatively easy retrieved 
from different biological databases, such as those mentioned
in the previous section. On the other hand, obtaining 
the structures of the macromolecules (enzymes) can be more 
challenging. This is because enzyme structures are often 
distributed across different sources and databases. 
In this section, the reported sources for obtaining the 
three-dimensional structures of CES1 and CES2 enzymes 
will be displayed.

\subsubsection{CES1 crystal structures}
 
Since 2003, different crystallographic structures of CES1 have been 
reported, as summarized in Table~\ref{tab2}. The first published 
structures involved CES1 complexes with ligands from several categories,
including metabolites of abuse drugs like cocaine and heroin (homatropine 
and naloxone methiodide, respectively)\cite{Bencharit2003}, 
therapeutic drugs (tacrine, tamoxifen and mevastatin)\cite{Bencharit2003a,
Fleming2005}, and endogenous substrates (cholate, taurocholate and coenzyme A)
\cite{Bencharit2006} (Table~\ref{tab2}). Notably, the crystal structure 
of CES1-tamoxifen complex revealed that tacrine interacts in the
catalytic binding site of CES1 in four different binding modes\cite{Bencharit2003a}.
These findings highlights the promiscuous nature of CES1, suggesting that its
ability to hydrolyzed a variety of substrates is driven by its capacity to interact 
with these ligands in multiple conformations\cite{Bencharit2003a}.

\begin{table}[H] 
	\small % Change table font size
	\caption{Information related to CES1 reported crystallographic structures.\label{tab2}}
	\isPreprints{\centering}{} % Only used for preprints
	\begin{tabularx}{\textwidth}{CCCCC}
		\toprule
		\textbf{Substrates}	& \textbf{Classification} & \textbf{Resolution (\AA)} & \textbf{PDB code} &  \textbf{Reference}\\
		\midrule
		Homatropine	& M.L. & 2.80 & 1MX5 & \cite{Bencharit2003}\\
		Naloxone methiodide	& M.L. &2.90 & 1MX9 & \cite{Bencharit2003}\\
		Tacrine	& Drug &2.40 & 1MX1 & \cite{Bencharit2003a}\\
		Tamoxifen & Drug &3.20 & 1YA4 & \cite{Fleming2005}\\
		Mevastatin & Drug &3.00 & 1YA8 & \cite{Fleming2005}\\
		Ethylacetate&M.L.& 3.00& 1YAH & \cite{Fleming2005}\\
		Benzil & M.L.&3.20& 1YAJ & \cite{Fleming2005}\\
		Cholate/Palmitate &E.S.& 3.00& 2DQY & \cite{Bencharit2006}\\
		CoenzymeA& E.S.&2.00 & 2H7C& \cite{Bencharit2006}\\
		CoenzymeA/Palmitate& E.S.&2.80 & 2DQZ& \cite{Bencharit2006}\\
		Taurocholate & E.S.&3.20& 2DR0 & \cite{Bencharit2006}\\
		Soman & N.A.& 2.70& 2HRQ & \cite{Fleming2007}\\
		Tabun & N.A.& 2.70& 2HRR & \cite{Fleming2007}\\
		Cyclosarin & N.A.& 3.10 & 3K9B & \cite{Hemmert2010}\\
		- & -& 2.20 & 4AB1 & \cite{Greenblatt2012}\\
		- & -& 1.86 & 5A7F & \cite{ArenadeSouza2015}\\
		- & -& 2.67 & 8EOR & \cite{Su2023}\\
		F-3 & C.I.& 1.83 & 9KWL &\cite{Gai2024}\\
		F-4 & C.I.& 1.89 & 9KWM &\cite{Gai2024}\\
		\bottomrule
	\end{tabularx}
	
	\noindent{\footnotesize{M.L.: Metabolite ligand. E.S.: Endogenous substrate.
			 N.A.: Nerve agent. C.I.: covalent inhibitor.}}
\end{table}
 
Different types of covalent ligands have also been studied through 
crystallographic structures. The first group to be investigated included
the organophosphorus nerve agents soman, tabun and cyclosarin
\cite{Fleming2007,Hemmert2010}. More recently, attention shifted to 
a second family focused on the covalent binding mechanism of 
serine-selective electrophilic warheads. Two crystal structure were
generated showing CES1 covalently bound to 2,2,2-triﬂuoroacetophenone 
derivatives at the catalytic serine (Table~\ref{tab2})\cite{Gai2024}. 
Additionally, there are crystallographic structures of CES1 in the
absence of substrates\cite{Greenblatt2012,ArenadeSouza2015,Su2023}. 
One of these structures exhibited the highest resolution of any 
reported CES1 crystal structure to date (1.86~\AA).

For conducting molecular modeling studies focused on substrate 
selectivity between CES isoforms, the optimal approach is to use 
the CES1 structure with the highest possible resolution and 
complexed with non-covalent ligands or in a substrate-free state. 
On the other hand, it is not advisable to use the crystal structures 
of CES1 bound to covalent ligands. This is because the catalytic residues, 
along with other residues in the catalytic binding site, are in 
conformations suitable for binding covalent inhibitors, rather than
for accommodating non-covalent substrates.

\subsubsection{CES2 homology modeling structures}

The crystallographic structure of CES2 remains unresolved to this
day. As a result, homology modeling is necessary to obtain 
the three-dimensional structure of this CES isoform in order to model
the binding mode of substrates in the catalytic site of the enzyme. 
Different research publications have reported distinct strategies for
generating the homology model structure of CES2. One of the most common 
methodologies involves using tolls provided by the Swiss Institute of
Bioinformatics server (\url{www.expasy.org/}). In the early publications,
the process of generating the CES2 model followed a two-steps procedure:
first, the amino acid sequence of human CES2 was retrieved from the Swiss-Prot
database, and then submitted to the Swiss-Model\cite{Waterhouse2018} server 
for fully automated protein structure homology modeling\cite{Vistoli2010a,
Zou2016,Song2019}. Over time, this methodology became more automated, 
allowing users to directly download pre-existing CES2 models created
by other researchers via the Swiss-Model server\cite{Qian2021,
Lv2025,Ribone2025}.

The second approach involves generating the CES2 structure using 
homology modeling software. Several studies have reported the use 
of the open-source \textit{Modeler} software\cite{Webb2017} to generate
the homology model of CES2 and perform subsequent molecular modeling 
studies on this isoform\cite{Wang2018,Choudhary2019}.

In recent years, \textit{AlphaFold2}, a powerful neural network-based model
methodology has been developed for predicting the three-dimensional 
structures of proteins\cite{Jumper2021}. Since its beginning in 2021, 
this machine learning approach has generated more than 200 million protein 
structures, all freely available for download from the AlphaFold database 
(\url{alphafold.ebi.ac.uk}). To the best of our knowledge, despite its accessibility
and remarkable accuracy, an AlphaFold-predicted human CES2 structure has not yet
been utilized for the molecular modeling of this enzyme with any substrate.

\subsection{Molecular modeling methods}

To analyzed the results from experimental enzymology studies of the CES-substrate
complexes, a series of molecular modeling methodologies can be employed. These 
computational techniques were selected to investigate, at an atomistic level,
the kinetic parameters of different substrates for both CES isoforms. This section
is divided in two subsections: 1) a series of molecular modeling approaches (molecular
docking, molecular dynamics and free-energy of interaction analysis) used 
to study the affinity constant (K\textsubscript{M}) and 2) hybrid QM/MM simulations
employed to examined the catalytic constant (\textit{k\textsubscript{cat}}).

\subsubsection{Molecular modeling methods to analyzed affinity constant - K\textsubscript{M}}

Every molecular modeling campaign begins with a molecular docking
procedure, aimed at identifying the substrate conformations that 
correspond to the lowest energy interactions with the residues of
the CES catalytic binding site. In the next stage, the CES-substrate
complex undergoes a molecular dynamics (MD) simulation to examine the
dynamic behavior and stability of the complex in the presence of water
and at physiological temperature. Finally, a free-energy of interaction 
analysis is performed based on the previously obtained CES-substrate
MD simulation data. A small subset of studies incorporate all three of 
these molecular modeling methods in the investigation of CES-substrate 
complexes. Generally, however, most studies rely solely on molecular 
docking to identify the optimal substrate conformation for interaction
within the CES catalytic binding site.

The first molecular docking studies investigating the selectivity of
CES1 and CES2 were conducted by the group of Vistolli et al.
\cite{Vistoli2010,Vistoli2010a}. In these works, 40 known CES substrates 
were subjected to molecular docking protocols using two crystallographic 
structures of CES1 (PDB codes: 1MX9 and 1YAJ) and a homology modeling
structure of CES2. In addition, different scoring functions were calculated
and correlated with the experimentally reported K\textsubscript{M} values
to derive a predictive model for substrate affinity toward both CES isoforms. 
In both cases, the results revealed a strong correlation with the calculated
lipophilic interaction score between the substrates and the CES catalytic 
binding site residues, confirming the crucial role of hydrophobic interactions,
primarily due to the abundance of apolar residues within the binding 
site\cite{Vistoli2010,Vistoli2010a}.

A combination of molecular docking and MD simulation studies have
been used to assist in the design of selective inhibitors of human 
CES2. These studies were focused on the design, synthesis and 
structure-activity relationship of glycyrrhetinic acid and benzofuranone 
derivatives\cite{Zou2016,Yang2023}. In the first work,
molecular docking was performed with the most active and selective 
glycyrrhetinic acid derivative against both CES isoforms, aiming to
understand its 1000-fold selectivity for CES2 over CES1\cite{Zou2016}.
The results showed that this derivative displayed more hydrogen bond 
interactions with CES2 catalytic binding site residues than with CES1.
In addition, the conformation adopted by the inhibitor in CES1 complex
was positioned farther from the catalytic serine residue compared to 
the CES2 complex\cite{Zou2016}. In the second study, a similar molecular 
docking analysis was conducted for the most active and selective 
benzofuranone derivative against both isoforms. The results were consistent,
with the inhibitor showing better interaction and higher binding site 
contacts with residues of CES2 than with CES1\cite{Yang2023}. Furthermore,
MD simulation and free-energy decomposition analyses were performed 
on the CES2-inhibitor complex, revealing that the complex remain stable 
after 50ns of MD simulation and that the hydrophobic interactions played
a key role in the binding of the inhibitor to CES2\cite{Yang2023}. 
Figure~\ref{fig4} exhibited that both selective CES2 inhibitors contain
a carboxylic acid group in their corresponding molecular structures.

\begin{figure}[H]
	%\isPreprints{}{% This command is only used for ``preprints''.
		\begin{adjustwidth}{-\extralength}{0cm}
			\centering
			%} % If the paper is ``preprints'', please uncomment this parenthesis.
		\subfloat[\centering]{\includegraphics[width=7.0cm]{Figure4a.png}}
		%\hfill
		\subfloat[\centering]{\includegraphics[width=7.5cm]{Figure4b.png}}\\
		%\isPreprints{}{% This command is only used for ``preprints''.
		\end{adjustwidth}
		%} % If the paper is ``preprints'', please uncomment this parenthesis.
	\caption{Molecular structure of CES2 selective inhibitors. 
		(\textbf{a}) Glycyrrhetinic acid derivative. (\textbf{b})  
		Benzofuranone derivative.\label{fig4}}
\end{figure} 


In a more recent study\cite{Ribone2025}, molecular docking, MD simulation
and free-energy interaction decomposition analyses were performed on two 
families of previosly studied substrates: five \textit{p}-nitrophenyl ester
derivatives\cite{Hatfield2010} and two pyrethroid stereoisomers\cite{Nishi2006},
using both CES1 and CES2. As in earlier studies, hydrophobic interactions 
were found to significantly contribute to the CES-ligand free-energy of 
interaction complex, correlating well with the experimental determined 
affinity constants (K\textsubscript{M})\cite{Ribone2025}. An additional key
observation was that the increase cavity volume of CES2, relative to CES1,
allowed one pyrethroid stereoisomer to exhibit a distinct interaction pattern,
thereby maintain a higher affinity (lower K\textsubscript{M}) for CES2 
compared to other stereoisomer. The studied substrates initially generate 
the best possible interaction pattern, with the residues of the
respective CES catalytic binding site, in order to produce the 
highest possible affinity constant with the corresponding CES isoform\cite{Ribone2025}.

\subsubsection{Hybrid QM/MM simulation to analyzed catalytic constant - \textit{k\textsubscript{cat}}}

The study of enzyme-ligand catalytic constant through 
computational chemistry requires a specialized methodology 
to analyzed the free-energy of activation involved in the
hydrolytic reaction to the ligand produced by the enzyme. The most
commonly used molecular modeling technique for this purposed is 
the hybrid QM/MM simulation. In this approach, the ligand and the
catalytic residues involved in the hydrolysis reaction are modeled 
using quantum mechanics (QM), while the remaining enzyme residues are
treated with a molecular mechanics (MM) force field.

The general hydrolysis mechanism of esterases, such as CES, has been 
well established and involves two consecutive reactions. First, 
the acylation step, where the carbonyl carbon of the substrate 
is attacked by the hydroxyl moiety from the catalytic serine, 
resulting in the formation of the acylated serine residue and the 
release the alkyl portion of the substrate. Second, the deacylation 
reaction, where the carbonyl carbon of the acylated serine is attacked
by the oxygen of a water molecule, yielding the caboxylic acid portion 
of the substrate and regenerating the free serine residue, thereby allowing
a new catalytic cycle to begin (Figure~\ref{fig5})\cite{Hosokawa2008,Yao2018,Ribone2025}.

\begin{figure}[H]
	%\isPreprints{\centering}{} % Only used for preprints
	\includegraphics[width=15.0 cm]{Figure5.png}
	\caption{Acylation and deacylation reaction pathways for CES1 and CES2 catalyzed hydrolysis of the ester ligands. \textit{p}-
	nitrophenyl ester derivatives were used as example. Reactant state (RS), first transition state (TS1), intermediate (INT), second transition state (TS2) and product state (PS).\label{fig5}}
\end{figure} 

There have not been many reported works involving the hybrid QM/MM
simulation between substrates hydrolyzed by human CES, probably
because of the higher complexity of this methodology compared with
the molecular modeling methodologies analyzed in the previous section.
Based on the importance of the metabolism of abuse drugs by human CES,
different works explored the hydrolysis of cocaine by CES1 and CES2.
The first article reported the hydrolysis mechanism of cocaine by CES1
using hybrid QM/MM simulation and also the experimental determination
of the corresponding kinetic data (K\textsubscript{M}
and \textit{k\textsubscript{cat}})\cite{Yao2018}. The QM region was
parameterized with the semi-empirical SCC-DFTB method and the MM
region with CHARMM27 force field. The study was based in an umbrella
sampling method where the distance of two covalent bond lengths,
a new covalent bond forming and a covalent bond breaking, were
combined linearly to generate the reaction coordinate (RC). Based 
on the results of the simulations, CES1 catalyzed cocaine hydrolysis
to benzoylecgonine and methanol (Figure\ref{fig1}) by a single-step
acylation stage follow by a one-step deacylation stage showing one transition state (TS) for every one of these reactions. The free-energy profile plotted by combination of both acylation and deacylation reactions displayed that the first TS, corresponding to the acylation stage, is the rate-limiting transition state with a free energy barrier of 
20.1 kcal/mol. The experimental free-energy barrier, calculated based
on the experimental catalytic constant, exhibited a value of 21.5
kcal/mol, demonstrating a high accuracy in the developed hybrid
QM/MM simulation methodology of this work\cite{Yao2018}.

A second article reported a similar study that the previous one,
exploring the hydrolysis of cocaine by CES2 to produced ecgonine 
methyl ester and benzoic acid (Figure~\ref{fig1}). In this study,
the QM region was parameterized with the semi-empirical PM6 method and 
the MM area with an Amber18 force field (ff99SB). An umbrella
sampling method was again in this case used to follow the two
steps hydrolysis reaction, exploring two covalent bond lengths,
one covalent bond forming and one covalent bond breaking. The 
calculated 2D-PMF profiles of the full catalytic cycle showed
that every reaction stage proceed with an tetrahedral intermediate
structure and two TS points with the TS\textsubscript{4} (formation
of benzoic acid) was identified as the rate-limiting step of the 
cocaine hydrolysis by the CES2, displaying an activation free-energy
barrier of 19.5 kcal/mol. This lower energetic barrier respect to 
CES1 is in agreement with previous experimental results that showed 
a higher turnover number for CES2 than CES1.

In the previous section, a work where two families of substrates,
\textit{p}-nitrophenyl esters and pyrethroid stereoisomers, in 
complex with CES1 and CES2 were reported. In this reported study,
two \textit{p}-nitrophenyl ester derivatives and the two pyrethroid
stereoisomers were submitted to hybrid QM/MM simulation to analyzed
the hydrolysis produced by both CES isoforms\cite{Ribone2025}. The
followed QM/MM simulation protocol was similar to the previous study
of cocaine and CES1\cite{Yao2018}, the QM region was parameterized 
with the same semi-empirical SCC-DFTB3 method and the MM region with Amber20 force field (ff14SB). As with the previous example, in this 
case the study was also based in an umbrella sampling method, but
in this study the distances of four covalent bond lengths, two 
covalent bonds forming and two covalent bonds breaking, were
combined linearly, following a linear combination of distances (LCOD),
to generate the reaction coordinate (RC). In all the modeled 
reactions, the hydrolysis of the substrates by both CES followed 
a single-step acylation stage follow by one-step deacylation stage, 
with one transition state (TS) for every one of these reactions.
The analysis of the results showed that the rate-limiting step for the studied substrates reactions were related to the TS with the higher 
steric hindrance molecular structure produced during the course of 
the hydrolysis, acylation and deacylation stage, reactions
\cite{Ribone2025}. The conclusion of this work was that the CES selectivity is not only related to the molecular size of the alkyl 
or acyl groups of the substrates, but a more complex scenario driven
by the initial conformation of the ligand in the interaction with
the CES binding site\cite{Ribone2025}.

In general, all the reported studies in this section showed good 
correlation between the experimental catalytic constant
(\textit{k\textsubscript{cat}}) and the calculated free-energy of
activation from the rate-limiting TS reaction stage, exhibiting
an accurate methodology of the hybrid QM/MM simulation studies,
despite the mentioned differences in the general hydrolysis 
mechanisms, displaying two or four TS structures.

\section{Part III: Combination of experimental and computational enzymology} 

This last section of the article is dedicated to the published works where
both experimental and computational methodologies were combined to
study the hydrolysis properties of different substrates by the different
CES isoforms.

In the first one, the study of fenofibrate hydrolysis in presence of
CES1 and CES2 was studied in order to identify the main enzyme responsible
for its metabolization in humans\cite{Li2023}. The kinetic parameters were obtained by analyzing the formation rate of fenoﬁbric acid (the hydrolytic metabolite of fenoﬁbrate) through HPLC-UV in presence of human microsoms and 
recombinant CES1 and CES2 enzymes. These results exhibited that the 
affinity constant is slightly higher for CES1, but the main difference
was present in the catalytic constant, where CES1 displayed a extensive
faster reaction rate than with CES2\cite{Li2023}. The authors performed
a molecular docking study of the substrate in the binding site of both
CES isoforms. This study showed that fenoﬁbrate exhibited a more favorable
docking score into the catalytic binding site of CES1 than CES2, similar
to the previous displayed experimental affinity results. Hybrid QM/MM
simulations were not performed in this publication\cite{Li2023}. This
study would be very helpful to understand the reason behind the high 
difference in the hydrolysis of fenofibrate between the CES isoforms.

The second study, reported the design and CES selectivity of a family
of four fluorescent substrates derivatives of naphthalimide structure
\cite{Jia2021}. The enzymatic assays were followed by measuring the 
emission intensity of the hydrolysis product at 520 nm (excitation 450 
nm) after incubation with CES1 and CES2. Only the two derivatives 
containing the amide and carbamate moieties in their structures showed
specific hydrolysis only in presence of CES2 (NIC-1 and NIC-2, 
figure~\ref{fig4}). The other two carbamates with the nitrogen bonded 
to all carbon and no hydrogen atoms, showed inhibitory effects over both CES isoforms (NIC-3 and NIC-4, figure~\ref{fig4}). In order to study the change  
in the enzymatic activity from CES substrate to inhibitor, NIC-4 binding properties were studied by molecular modeling methods with CES1 and 
compared with the known substrate \textit{p}-nitrophenyl acetate (pNPA)
\cite{Jia2021}. After molecular docking and MD simulations, the binding free-energy of interaction analysis displayed that both NIC-4 and pNPA 
exhibited similar pattern of interaction with the residues of CES1 
binding site. In addition, a steered MD simulation was performed, where
the structures of the substrates and the binding site residues were 
parameterized with SCC-DFTB method and the remaining part of the enzyme
with Amber16 ff14SB force ﬁeld. The reaction coordinate was set as the 
distance from the catalytic serine oxygen to the carbonyl carbon of the
substrates, representing the nucleophilic attack in the hydrolysis reaction.
During the simulation of the hydrolysis reaction for pNPA with CES1, the 
proton from the catalytic serine residue is automatically transfer to 
the nitrogen atom in the catalytic histidine, following the normal acylation
mechanism with an estimated energetic barrier of 20 kcal/mol. 
In the other hand, in the simulation of NIC-4 with CES1, such proton transfer would not happen because the methyl group near the potential nucleophilic center of NIC-4 (Figure~\ref{fig6}) hinders the transfer path and forces the hydroxyl group of serine to orient away from the imidazole group of histidine. These 
molecular modeling results helped the authors to explained the mechanism of
CES inhibition produced by NIC-4\cite{Jia2021}.


\begin{figure}[H]
	\includegraphics[width=10.0 cm]{Figure6.png}
	\caption{Structure of the fluorescent substrates derivatives of naphthalimide (NIC)\cite{Jia2021}.\label{fig6}}
\end{figure}

\section{Conclusions}

This section is not mandatory, but can be added to the manuscript if the discussion is unusually long or complex.



%\begin{table}[H] 
%\small % Change table font size
%\caption{This is a table caption. Tables should be placed in the main text near to the first time they are~cited.\label{tab1}}
%\isPreprints{\centering}{} % Only used for preprints
%\begin{tabularx}{\textwidth}{CCC}
%\toprule
%\textbf{Substrates}	& \multicolumn{2}\textbf{}	& \textbf{Title 3}\\
%\midrule
%Entry 1		& Data			& Data\\
%Entry 2		& Data			& Data \textsuperscript{1}\\
%\bottomrule
%\end{tabularx}

%\noindent{\footnotesize{\textsuperscript{1} Tables may have a footer.}}
%\end{table}



% Example of a figure that spans the whole page width and with subfigures. The same concept works for tables, too.
\begin{figure}[H]
%\isPreprints{}{% This command is only used for ``preprints''.
\begin{adjustwidth}{-\extralength}{0cm}
\centering
%} % If the paper is ``preprints'', please uncomment this parenthesis.
\subfloat[\centering]{\includegraphics[width=7.0cm]{Definitions/logo-mdpi}}
%\hfill
\subfloat[\centering]{\includegraphics[width=7.0cm]{Definitions/logo-mdpi}}\\
\subfloat[\centering]{\includegraphics[width=7.0cm]{Definitions/logo-mdpi}}
%\hfill
\subfloat[\centering]{\includegraphics[width=7.0cm]{Definitions/logo-mdpi}}
%\isPreprints{}{% This command is only used for ``preprints''.
\end{adjustwidth}
%} % If the paper is ``preprints'', please uncomment this parenthesis.
\caption{This is a wide figure. Schemes follow the same formatting. If there are multiple panels, they should be listed as: (\textbf{a}) Description of what is contained in the first panel. (\textbf{b}) Description of what is contained in the second panel. (\textbf{c}) Description of what is contained in the third panel. (\textbf{d}) Description of what is contained in the fourth panel. Figures should be placed in the main text near to the first time they are cited. A caption on a single line should be centered.\label{fig22}}
\end{figure} 

\begin{table}[H]
\caption{This is a wide table.\label{tab22}}
%\isPreprints{\centering}{% This command is only used for ``preprints''.
	\begin{adjustwidth}{-\extralength}{0cm}
%} % If the paper is ``preprints'', please uncomment this parenthesis.
%\isPreprints{\begin{tabularx}{\textwidth}{CCCC}}{% This command is only used for ``preprints''.
		\begin{tabularx}{\fulllength}{CCCC}
%} % If the paper is ``preprints'', please uncomment this parenthesis.
			\toprule
			\textbf{Title 1}	& \textbf{Title 2}	& \textbf{Title 3}     & \textbf{Title 4}\\
			\midrule
\multirow[m]{3}{*}{Entry 1 *}	& Data			& Data			& Data\\
			  	                   & Data			& Data			& Data\\
			             	      & Data			& Data			& Data\\
                   \midrule
\multirow[m]{3}{*}{Entry 2}    & Data			& Data			& Data\\
			  	                  & Data			& Data			& Data\\
			             	     & Data			& Data			& Data\\
			\bottomrule
		\end{tabularx}
%		\isPreprints{}{% This command is only used for ``preprints''.
	\end{adjustwidth}
%} % If the paper is ``preprints'', please uncomment this parenthesis.
	\noindent{\footnotesize{* Tables may have a footer.}}
\end{table}

%\begin{listing}[H]
%\caption{Title of the listing}
%\rule{\columnwidth}{1pt}
%\raggedright Text of the listing. In font size footnotesize, small, or normalsize. Preferred format: left aligned and single spaced. Preferred border format: top border line and bottom border line.
%\rule{\columnwidth}{1pt}
%\end{listing}



%% Example of a page in landscape format (with table and table footnote).
%\startlandscape
%\begin{table}[H] %% Table in wide page
%%\isPreprints{\centering}{} % This command is only used for ``preprints''.
%\caption{This is a very wide table.\label{tab3}}
%	\begin{tabularx}{\textwidth}{CCCC}
%		\toprule
%		\textbf{Title 1}	& \textbf{Title 2}	& \textbf{Title 3}	& \textbf{Title 4}\\
%		\midrule
%		Entry 1		& Data			& Data			& This cell has some longer content that runs over two lines.\\
%		Entry 2		& Data			& Data			& Data\textsuperscript{1}\\
%		\bottomrule
%	\end{tabularx}
%%\isPreprints{}{% This command is only used for ``preprints''.
%	\begin{adjustwidth}{+\extralength}{0cm}
%%} % If the paper is ``preprints'', please uncomment this parenthesis.
%		\noindent\footnotesize{\textsuperscript{1} This is a table footnote.}
%%\isPreprints{}{% This command is only used for ``preprints''.
%	\end{adjustwidth}
%%} % If the paper is ``preprints'', please uncomment this parenthesis.
%\end{table}
%\finishlandscape




%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Discussion}

Authors should discuss the results and how they can be interpreted from the perspective of previous studies and of the working hypotheses. The findings and their implications should be discussed in the broadest context possible. Future research directions may also be highlighted.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Conclusions}

This section is not mandatory, but can be added to the manuscript if the discussion is unusually long or complex.


\vspace{6pt} 

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% optional
%\supplementary{The following supporting information can be downloaded at:  \linksupplementary{s1}, Figure S1: title; Table S1: title; Video S1: title.}

% Only for journal Methods and Protocols:
% If you wish to submit a video article, please do so with any other supplementary material.
% \supplementary{The following supporting information can be downloaded at: \linksupplementary{s1}, Figure S1: title; Table S1: title; Video S1: title. A supporting video article is available at doi: link.}

% Only used for preprtints:
% \supplementary{The following supporting information can be downloaded at the website of this paper posted on \href{https://www.preprints.org/}{Preprints.org}.}

% Only for journal Hardware:
% If you wish to submit a video article, please do so with any other supplementary material.
% \supplementary{The following supporting information can be downloaded at: \linksupplementary{s1}, Figure S1: title; Table S1: title; Video S1: title.\vspace{6pt}\\
%\begin{tabularx}{\textwidth}{lll}
%\toprule
%\textbf{Name} & \textbf{Type} & \textbf{Description} \\
%\midrule
%S1 & Python script (.py) & Script of python source code used in XX \\
%S2 & Text (.txt) & Script of modelling code used to make Figure X \\
%S3 & Text (.txt) & Raw data from experiment X \\
%S4 & Video (.mp4) & Video demonstrating the hardware in use \\
%... & ... & ... \\
%\bottomrule
%\end{tabularx}
%}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\authorcontributions{For research articles with several authors, a short paragraph specifying their individual contributions must be provided. The following statements should be used ``Conceptualization, X.X. and Y.Y.; methodology, X.X.; software, X.X.; validation, X.X., Y.Y. and Z.Z.; formal analysis, X.X.; investigation, X.X.; resources, X.X.; data curation, X.X.; writing---original draft preparation, X.X.; writing---review and editing, X.X.; visualization, X.X.; supervision, X.X.; project administration, X.X.; funding acquisition, Y.Y. All authors have read and agreed to the published version of the manuscript.'', please turn to the  \href{http://img.mdpi.org/data/contributor-role-instruction.pdf}{CRediT taxonomy} for the term explanation. Authorship must be limited to those who have contributed substantially to the work~reported.}

\funding{Please add: ``This research received no external funding'' or ``This research was funded by NAME OF FUNDER grant number XXX.'' and  and ``The APC was funded by XXX''. Check carefully that the details given are accurate and use the standard spelling of funding agency names at \url{https://search.crossref.org/funding}, any errors may affect your future funding.}

\institutionalreview{In this section, you should add the Institutional Review Board Statement and approval number, if relevant to your study. You might choose to exclude this statement if the study did not require ethical approval. Please note that the Editorial Office might ask you for further information. Please add “The study was conducted in accordance with the Declaration of Helsinki, and approved by the Institutional Review Board (or Ethics Committee) of NAME OF INSTITUTE (protocol code XXX and date of approval).” for studies involving humans. OR “The animal study protocol was approved by the Institutional Review Board (or Ethics Committee) of NAME OF INSTITUTE (protocol code XXX and date of approval).” for studies involving animals. OR “Ethical review and approval were waived for this study due to REASON (please provide a detailed justification).” OR “Not applicable” for studies not involving humans or animals.}

\informedconsent{Any research article describing a study involving humans should contain this statement. Please add ``Informed consent was obtained from all subjects involved in the study.'' OR ``Patient consent was waived due to REASON (please provide a detailed justification).'' OR ``Not applicable'' for studies not involving humans. You might also choose to exclude this statement if the study did not involve humans.

Written informed consent for publication must be obtained from participating patients who can be identified (including by the patients themselves). Please state ``Written informed consent has been obtained from the patient(s) to publish this paper'' if applicable.}

\dataavailability{We encourage all authors of articles published in MDPI journals to share their research data. In this section, please provide details regarding where data supporting reported results can be found, including links to publicly archived datasets analyzed or generated during the study. Where no new data were created, or where data is unavailable due to privacy or ethical restrictions, a statement is still required. Suggested Data Availability Statements are available in section ``MDPI Research Data Policies'' at \url{https://www.mdpi.com/ethics}.} 



\acknowledgments{In this section you can acknowledge any support given which is not covered by the author contribution or funding sections. This may include administrative and technical support, or donations in kind (e.g., materials used for experiments). Where GenAI has been used for purposes such as generating text, data, or graphics, or for study design, data collection, analysis, or interpretation of data, please add “During the preparation of this manuscript/study, the author(s) used [tool name, version information] for the purposes of [description of use]. The authors have reviewed and edited the output and take full responsibility for the content of this publication.”}

\conflictsofinterest{Declare conflicts of interest or state ``The authors declare no conflicts of interest.'' Authors must identify and declare any personal circumstances or interest that may be perceived as inappropriately influencing the representation or interpretation of reported research results. Any role of the funders in the design of the study; in the collection, analyses or interpretation of data; in the writing of the manuscript; or in the decision to publish the results must be declared in this section. If there is no role, please state ``The funders had no role in the design of the study; in the collection, analyses, or interpretation of data; in the writing of the manuscript; or in the decision to publish the results''.} 

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Optional

%% Only for journal Encyclopedia
%\entrylink{The Link to this entry published on the encyclopedia platform.}

\abbreviations{Abbreviations}{
The following abbreviations are used in this manuscript:
\\

\noindent 
\begin{tabular}{@{}ll}
MDPI & Multidisciplinary Digital Publishing Institute\\
DOAJ & Directory of open access journals\\
TLA & Three letter acronym\\
LD & Linear dichroism
\end{tabular}
}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\isPreprints{}{% This command is only used for ``preprints''.
\begin{adjustwidth}{-\extralength}{0cm}
%} % If the paper is ``preprints'', please uncomment this parenthesis.
%\printendnotes[custom] % Un-comment to print a list of endnotes

\reftitle{References}

% Please provide either the correct journal abbreviation (e.g. according to the “List of Title Word Abbreviations” http://www.issn.org/services/online-services/access-to-the-ltwa/) or the full name of the journal.
% Citations and References in Supplementary files are permitted provided that they also appear in the reference list here. 

%=====================================
% References, variant A: external bibliography
%=====================================
\externalbibliography{yes}
\bibliography{JoX-2025}
%\bibliographystyle{mdpi}
%=====================================
% References, variant B: internal bibliography
%=====================================


% If authors have biography, please use the format below
%\section*{Short Biography of Authors}
%\bio
%{\raisebox{-0.35cm}{\includegraphics[width=3.5cm,height=5.3cm,clip,keepaspectratio]{Definitions/author1.pdf}}}
%{\textbf{Firstname Lastname} Biography of first author}
%
%\bio
%{\raisebox{-0.35cm}{\includegraphics[width=3.5cm,height=5.3cm,clip,keepaspectratio]{Definitions/author2.jpg}}}
%{\textbf{Firstname Lastname} Biography of second author}

% For the MDPI journals use author-date citation, please follow the formatting guidelines on http://www.mdpi.com/authors/references
% To cite two works by the same author: \citeauthor{ref-journal-1a} (\citeyear{ref-journal-1a}, \citeyear{ref-journal-1b}). This produces: Whittaker (1967, 1975)
% To cite two works by the same author with specific pages: \citeauthor{ref-journal-3a} (\citeyear{ref-journal-3a}, p. 328; \citeyear{ref-journal-3b}, p.475). This produces: Wong (1999, p. 328; 2000, p. 475)

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% for journal Sci
%\reviewreports{\\
%Reviewer 1 comments and authors’ response\\
%Reviewer 2 comments and authors’ response\\
%Reviewer 3 comments and authors’ response
%}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\PublishersNote{}
%\isPreprints{}{% This command is only used for ``preprints''.
\end{adjustwidth}
%} % If the paper is ``preprints'', please uncomment this parenthesis.
\end{document}

